app/schemas/com.keylesspalace.tusky.db.AppDatabase/39.json

```diff
@@ -2,11 +2,11 @@
   "formatVersion": 1,
   "database": {
     "version": 39,
-    "identityHash": "ed3b752a3faec9d092d5ac0a2823d5d5",
+    "identityHash": "7776f4c93b76990df18a3f8f3bcbb9bb",
     "entities": [
       {
         "tableName": "DraftEntity",
-        "createSql": "CREATE TABLE IF NOT EXISTS `${TABLE_NAME}` (`id` INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL, `accountId` INTEGER NOT NULL, `inReplyToId` TEXT, `content` TEXT, `contentWarning` TEXT, `sensitive` INTEGER NOT NULL, `visibility` INTEGER NOT NULL, `attachments` TEXT NOT NULL, `poll` TEXT, `failedToSend` INTEGER NOT NULL)",
+        "createSql": "CREATE TABLE IF NOT EXISTS `${TABLE_NAME}` (`id` INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL, `accountId` INTEGER NOT NULL, `inReplyToId` TEXT, `content` TEXT, `contentWarning` TEXT, `sensitive` INTEGER NOT NULL, `visibility` INTEGER NOT NULL, `attachments` TEXT NOT NULL, `poll` TEXT, `failedToSend` INTEGER NOT NULL, `scheduledAt` TEXT)",
```

Comment:
@connyduck Is this harmless? why did room change 39? Should I remove this change? I think this was part of my "original" change, because I added the column when the DB was at 39 (and *I* was introducing 40). With the merge and me moving to v41, I'm not sure if this would have any effect during a migration. What do you think?

Reply:
THanks for clarifying. Reverted; sorry for the trouble.